File: twoSumII/tsii.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] res = new int[2];
        Map<Integer,Integer> map = new HashMap<Integer,Integer>();
        for(int i = 0;i<nums.length;i++){
            if(map.get(target - nums[i])!=null){
                res[0] = map.get(target - nums[i])+1;
                res[1] = i+1;
                return res;
            }
            map.put(nums[i],i);
        }
        return null;
    }
}

File: twoSumII/tsii.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        dic = {}
        for i,num in enumerate(numbers):
            if target-numbers[i] in dic:
                return [dic[target-numbers[i]]+1,i+1]
            dic[num] = i 
        return None